app/views/spaces/_static_map.html.erb

```diff
@@ -0,0 +1,23 @@
+<%
+  mapbox_zoom = 12
+  map_height = 250
+  map_width = 400
+  mapbox_static_map_url = %W[
+      https://api.mapbox.com/styles/v1/mapbox/streets-v11/static
+      /pin-l-circle+db2777(#{@space.lng},#{@space.lat})
+      /#{@space.lng},#{@space.lat},#{mapbox_zoom}
+      /#{map_width}x#{map_height}?logo=false&@2x&
+      &access_token=#{ENV["MAPBOX_API_KEY"]}
+  ].join
+
+  google_maps_url = %W[
+    https://www.google.com/maps?q=
+    #{ERB::Util.url_encode(
+      "#{@space.title}, #{@space.address}, #{@space.post_number} #{@space.post_address}"
+    )}
+  ].join
```

Comment:
Can we move this out of the view file and into a helper? Looks like too much logic for a view file

Reply:
We definitely could. 

All the logic is just to render this one map though. Should that be in the controller, or should it be in the view file? Not sure what the standard is here!

Reply:
Agree. Maybe put it in a helper.